docs/develop/dotnet/durable-timers.mdx

```diff
@@ -0,0 +1,30 @@
+---
+id: durable-timers
+title: Durable Timers - .NET SDK feature guide
+description: Set a Durable Timer to sleep for days, weeks, or years in a Workflow.
+sidebar_label: Durable Timers
+keywords:
+  - sdk
+  - dotnet
+  - durable timers
+  - sleep
+  - time-skipping
+tags:
+  - sdk
+  - dotnet
+  - durable-timers
+  - sleep
+  - time-skipping
+---
+
+## Sleep
+
+**How to sleep in a Workflow using the Temporal .NET SDK**
+
+<!-- CONTENT-->
+
+## Skip time in tests {#testing-skip-time}
```

Comment:
I don't think this should be here, I think it should be with the testing documentation